homeassistant/components/onvif/camera.py

```diff
@@ -221,3 +221,23 @@ async def async_perform_ptz(
             tilt,
             zoom,
         )
+
+    async def async_run_aux_command(
```

Comment:
We don't need these additions to the camera platform now, right?

Reply:
that's correct, thanks. Removing now.